test/scripts/e2e-kind.sh

```diff
@@ -125,6 +124,13 @@ if [ "$DUALSTACK_CONVERSION" == true ]; then
   SKIPPED_TESTS=$SKIPPED_TESTS$DUALSTACK_CONVERSION_TESTS
 fi
 
+if [ "$OVN_GATEWAY_MODE" == "local" ]; then
+  if [ "$SKIPPED_TESTS" != "" ]; then
+    SKIPPED_TESTS+="|"
+  fi
+  SKIPPED_TESTS+="should fallback to local terminating endpoints when there are no ready endpoints with externalTrafficPolicy=Local"
+fi
+
```

Comment:
So I skipped this test in June 2022 because we didn't implement this feature, but this year I actually added support for it: https://github.com/ricky-rav/ovn-kubernetes/blob/master/go-controller/pkg/node/healthcheck_service.go#L163 :) 

I can have a look at what's wrong with LGW

Reply:
lgw has a more complicated way of doing etp=local...